LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
This LICENSE should probably be thought through. Since this is Finn data etc. Maybe it's better to have no license until we've talked to someone who has an opinion on that?

Reply:
Yes. We are allowed to share and use. So the most open license that allows for fair use (research, not unintended use such as reidentifcation?) and with attribution?